### DISCUSSION TITLE
Another Vercel account is using this domain

### DISCUSSION THREAD

**Discussion by riungemaina** [vercel/community#3983] [Help] [answered]
### Summary

Hello, I'm trying to set up this doman; castynet.africa and I get an error that its in use by another vercel account, I'm almost certain that that is a mistake, I had tried to move it to vercel a few weeks ago but failed, now I want to change the name servers to vercel ones but I dont get that option. I own the domain, so if someone else added it to their account by mistake could you clear that up?
I have been able to use Vercel's name server's in the past and I'm looking to do the same. 

### Steps to Reproduce

![Screenshot 2023-09-08 at 23 51 42](https://github.com/vercel/community/assets/51113345/8d1b43b4-22b2-46f7-bf03-ab59fc4973b6)

**Comment by amyegan** [comment, marked as answer]
Hey @riungemaina. If you add the recommended TXT record, that will verify your ownership of the domain so it can be used in your project.

**Reply by yaslam123** [reply]
hey @amyegan am facing the same problem but even after i entered the recommended TXT record its still not verifying 
<img width="1440" alt="Screenshot 2024-04-02 at 10 40 34 PM" src="https://github.com/vercel/community/assets/114346991/808fdfce-2388-4939-9ed0-fa6f298413fd">
<img width="1440" alt="Screenshot 2024-04-02 at 10 40 28 PM" src="https://github.com/vercel/community/assets/114346991/d6dcffe3-b1d1-4a25-8063-cba608eee48e">

**Comment by riungemaina** [comment]
Okay, thank you

**Reply by alvinarkansas** [reply]
Is it working @riungemaina? Because I'm facing the same issue here, I already set the TXT record as suggested (same with @yaslam123)
cc: @amyegan

**Reply by riungemaina** [reply]
I've just checked and I added the .txt file, the domain got verified but I still use my domain registrars name servers instead of Vercel ones. 
I think it might have been a time thing where I just had to wait a bit maybe but I didn't have the patience so

**Reply by amyegan** [reply]
@alvinarkansas It's also possible that your account was recently updated to our new account structure. So you might need to move your custom domain from your personal account over to your new hobby team. 

You can learn more about the account update in https://github.com/vercel/community/discussions/6298

**Reply by alvinarkansas** [reply]
Correct, I moved my domain to the new hobby team following [this instruction](https://vercel.com/docs/projects/domains/transfer-your-domain) and it works. Thanks @amyegan

**Reply by OwaisSiddiqui** [reply]
Yes, I noticed the domain was part of my personal account and could update the records there (from #6298):

> Your domains are still associated with your personal account, so you won't see them listed when you click on the team "Domains" tab near the top of the dashboard. Instead, you should click on your profile picture in the top right corner, click "Settings" in the dropdown panel, then click "[Domains](https://vercel.com/account/domains)" on the left side navigation list. From there you should be able to see your domains and move them to your new team using these steps: https://vercel.com/docs/projects/domains/transfer-your-domain
